ste/sender-blockBlob.go

```diff
@@ -192,6 +194,10 @@ func (s *blockBlobSenderBase) Prologue(ps common.PrologueState) (destinationModi
 	if s.jptm.ShouldInferContentType() {
 		s.headersToApply.BlobContentType = ps.GetInferredContentType(s.jptm)
 	}
+	if s.jptm.DeleteUncommittedBlocks() {
+		s.DeleteDstBlob()
```

Comment:
if this is called in the Prologue, is there any way we could call this more than once? If not, the sync.Once is not necessary